docker/docker-compose.dev.yaml

```diff
@@ -0,0 +1,37 @@
+version: '3.8'
+
+services:
+  mongodb:
+    image: mongo:latest
+    ports:
+      - 27017:27017
+    volumes:
+      - mongodb-data:/data/db
+  
+  redis-stack-server:
+    image: redis/redis-stack-server:latest
+    ports:
+      - 6379:6379
+    volumes:
+      - redis-data:/data/redis 
+
+  talawa-api-dev-container:
+    build:
+      context: .
```

Comment:
Context shouldn't be `..` instead of `.` ?

Reply:
noted